sbnana/CAFAna/Core/Cut.h

```diff
@@ -9,7 +9,7 @@
 
 #include "sbnana/CAFAna/Core/Var.h"
 
-namespace caf{class StandardRecord; class SRSpill; class SRSpillTruthBranch; class SRSlice;}
+namespace caf{class StandardRecord; class SRSpill; class SRTrueInteraction; class SRSlice;}
```

Comment:
Double checking, the `-` part of the diff here (`SRSpillTruthBranch`) is just an old, unused class right? I think that's what I recall.

Reply:
Yes, I see no place where `SRSpillTruthBranch` is used